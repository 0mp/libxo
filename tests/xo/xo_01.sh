#
# $Id$
#
# Copyright 2014, Juniper Networks, Inc.
# All rights reserved.
# This SOFTWARE is licensed under the LICENSE provided in the
# ../Copyright file. By downloading, installing, copying, or otherwise
# using the SOFTWARE, you agree to be bound by the terms of that
# LICENSE.

XO=$1
shift

XOP="${XO} --depth 1"

${XO} --open top

NF=
for i in one two three four; do
    ${XOP} ${NF} --wrap item 'Item {k:name} is {:value}\n' $i $i
    NF=--not-first
done

${XO} --close top