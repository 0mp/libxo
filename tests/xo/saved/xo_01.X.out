<top><item><name>one</name><value>one</value></item><item><name>two</name><value>two</value></item><item><name>three</name><value>three</value></item><item><name>four</name><value>four</value></item></top>